### ISSUE TITLE
Transmutation Interface crashes the game when it receives an item

### ISSUE THREAD

**Issue by James103** [DonovanDMC/ProjectExpansion#55]
To reproduce:
1. Install the mods listed below on a new instance (Minecraft 1.18.2, Forge 40.1.31).
2. Create a new world with cheats enabled.
3. Give yourself a Transmutation Interface and a Hopper.
4. Place the Transmutation Interface.
5. Place the Hopper next to and pointing into the Transmutation Interface.
6. Place any item (with an EMC value) in the hopper.
7. Crash

Logs: https://paste.ee/p/KaUIY

Modlist from crash report:
```
	Mod List: 
		client-1.18.2-20220404.173914-srg.jar             |Minecraft                     |minecraft                     |1.18.2              |DONE      |Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		ProjectE-1.18.2-PE1.0.1.jar                       |ProjectE                      |projecte                      |1.0.1               |DONE      |Manifest: NOSIGNATURE
		forge-1.18.2-40.1.31-universal.jar                |Forge                         |forge                         |40.1.31             |DONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		ProjectExpansion-1.18.2-1.0.6.jar                 |Project Expansion             |projectexpansion              |1.18.2-1.0.6        |DONE      |Manifest: e9:fa:82:d2:7b:66:ce:37:db:52:70:63:25:68:04:a9:72:41:8f:cf:6c:9a:da:52:ab:05:5b:97:2c:13:54:37
```

**Comment by DonovanDMC** [comment]
Confirmed for 1.18, fix in the works

**Comment by DonovanDMC** [comment]
Fixed in [1.0.7](https://github.com/DonovanDMC/ProjectExpansion/releases/tag/1.18.2-1.0.7)
